src/Detector/Hugo/HugoDetector.cs

```diff
@@ -66,31 +66,43 @@ private bool IsHugoApp(ISourceRepo sourceRepo, out string appDirectory)
             appDirectory = string.Empty;
 
             // Search for config.toml
```

Comment:
nit: let's update the comment before each if-statement so it accurately reflects what we're searching for (_i.e._, "Search for supported .toml file").

This comment applies for the other instances of this below.